site/index.html

```diff
@@ -1,74 +1,69 @@
 ---
 layout: default
 ---
-    <div class="container">
-      <div class="jumbotron">
+<div class="container">
+    <div class="jumbotron">
         <h1>Apache Arrow</h1>
         <p class="lead">Powering Columnar In-Memory Analytics</p>
         <p>
-          <a class="btn btn-lg btn-success" href="mailto:dev-subscribe@arrow.apache.org" role="button">Join Mailing List</a>
-          <a class="btn btn-lg btn-primary" href="{{ site.baseurl }}/install/" role="button">Install (0.8.0 Release - December 18, 2017)</a>
+            <a class="btn btn-lg btn-success" href="mailto:dev-subscribe@arrow.apache.org" role="button">Join Mailing List</a>
+            <a class="btn btn-lg btn-primary" href="{{ site.baseurl }}/install/" role="button">Install ({{site.data.versions['current'].number}} Release - {{site.data.versions['current'].date}})</a>
         </p>
-      </div>
-      <h4><a href="{{ site.baseurl }}/blog/"><strong>See Latest News</strong></a></h4>
-      <div class="row">
+    </div>
+    <h4><a href="{{ site.baseurl }}/blog/"><strong>See Latest News</strong></a></h4>
+    <div class="row">
         <div class="col-lg-4">
-          <h2>Fast</h2>
-          <p>Apache Arrow&#8482; enables execution engines to take advantage of
- the latest SIMD (Single input multiple data) operations included in modern
- processors, for native vectorized optimization of analytical data
- processing. Columnar layout is optimized for data locality for better
- performance on modern hardware like CPUs and GPUs.</p>
+            <p>Apache Arrow is an open source development platform for columnar in-memory data structures and processing. It defines a language-independent physical memory layout, enabling zero-copy, zero-deserialization interchange of flat and nested columnar data amongst a variety of systems such as Python, R, Apache Spark, ODBC protocols, and proprietary systems that utilize the open source components.</p>
```

Comment:
We'll need to do a little bit of work on this, as this isn't the top-level pitch that was discussed most recently on the dev@ mailing list. 

Here was that pitch

> Apache Arrow is a cross-language development platform for in-memory data. It
specifies a standardized language-independent columnar memory format for flat
and hierarchical data, organized for efficient analytic operations on modern
hardware. It also provides computational libraries and zero-copy streaming
messaging and interprocess communication. Languages currently supported include
C, C++, Java, JavaScript, Python, and Ruby.

Reply:
Got it. I've added a new commit with this language.